incubator-tools/amount_rectification_from_words_to_numbers/amount_rectification_from_words_to_numbers.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #454.            new_entity = documentai.Document.Entity()
</pre><p>Add the parameters when the entity is created, not after. It's more difficult to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
In this step , we are checking if there is an entity predicted , if not we are creating an entity and assigning  the values.